content/applications/inventory_and_mrp/quality/quality_check_types/picture_check.rst

```diff
@@ -124,30 +125,44 @@ specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a
 specified, a *Take a Picture* quality check is created for that specific work order, rather than the
 manufacturing order as a whole.
 
-*Take a Picture* quality checks created for work orders must be completed from the tablet view. To
-do so, begin by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing
-Orders`. Select a manufacturing order that includes a work order for which a quality check is
-required. Open the tablet view for that work order by selecting the :guilabel:`Work Orders` tab, and
-then clicking the :guilabel:`📱 (tablet)` button on the order's line.
+*Take a Picture* quality checks configured for work orders must be completed from the *Shop Floor*
+module. To do so, begin by navigating to :menuselection:`Manufacturing --> Operations -->
+Manufacturing Orders`. Select an |MO| that includes a work order for which a *Take a Picture*
+quality check is required.
 
-With tablet view open, complete the steps listed on the left side of the screen until the *Take a
-Picture* quality check step is reached. Upon reaching the check, follow the instructions for how to
-take the picture, which appear at the top of the screen.
+On the |MO|, select the :guilabel:`Work Orders` tab, and then click the :guilabel:`Open Work Order
+(square with arrow coming out of it)` button on the line of the work order to be processed. On the
+resulting :guilabel:`Work Orders` pop-up window, click the :guilabel:`Open Shop Floor` button to
+open the *Shop Floor* module.
```

Comment:
I'm not 100% sure what's going on with today's runbot. 

I **know** it worked like you have written before, but look at what happened when i tested it in runbot today ☠️☠️☠️

After clicking on the square with arrow coming out of it, I saw the instructions of the quality check in the pop-up window. I could not find the "Open Shop floor" button. 

![image](https://github.com/odoo/documentation/assets/23611711/565949c8-b126-4d99-9cba-d7580f12a86d)

And, when I clicked each line of the quality checks in the "Instructions" tab, it'd open the quality check in a new window.

![image](https://github.com/odoo/documentation/assets/23611711/94dc8619-9395-4e87-b365-d482b8826bba)

Did I set it up incorrectly?

Reply:
ok nvm this works in the 16.4 runbot !!